overlord/boot/firstboot.go

```diff
@@ -137,6 +139,27 @@ func populateStateFromSeed() error {
 	return ovld.Stop()
 }
 
+func readAsserts(fn string) ([]asserts.Assertion, error) {
+	res := ([]asserts.Assertion)(nil)
```

Comment:
The usual spelling here is
```
var res []asserts.Assertion
```
Might even just name the return variables (`(res ..., err error)`).